docs/3.0rc/api-ref/rest-api/server/schema.json

```diff
@@ -22279,6 +22279,11 @@
                         "title": "Prefect Task Scheduling Pending Task Timeout",
                         "default": "PT0S"
                     },
+                    "PREFECT_EXPERIMENTAL_ENABLE_EXTRA_RUNNER_ENDPOINTS": {
+                        "type": "boolean",
+                        "title": "Prefect Experimental Enable Extra Runner Endpoints",
+                        "default": false
+                    },
```

Comment:
i think you may want to remove this, as this appears to be why the statis analysis is failing